lib/l10n/app_de.arb

```diff
@@ -12,5 +18,20 @@
     "pageAccountThemesLight": "Hell",
     "pageAccountThemesDark": "Dunkel",
 
+    "driveChatPageTitle": "Fahrt Chat",
```

Comment:
the system I wanted to use was basically "{place in widget hierarchy} {object name} {category} {key}" if you look above. This should read the best because it goes top-down from most generic to most specific.

We should at least be consistent there.

Reply:
so how should that be changed i don't fully understand

Reply:
What it means is basically just put the "page", "card", ... part in front

Reply:
ok thanks